kornia/geometry/transform/pyramid.py

```diff
@@ -350,3 +350,49 @@ def build_pyramid(
         pyramid.append(img_down)
 
     return pyramid
+
+
+def build_laplacian_pyramid(
+    input: torch.Tensor, max_level: int, border_type: str = 'reflect', align_corners: bool = False
+) -> List[torch.Tensor]:
+    r"""Construct the Laplacian pyramid for an image.
+
+    .. image:: _static/img/build_pyramid.png
```

Comment:
to make this happen, you need to update this script
https://github.com/kornia/kornia/blob/master/docs/generate_example_images.py